### PR TITLE
cloud dm: add seoul region

### PR DESCRIPTION
**Pull request by hfxsd**
### What is changed, added or deleted? (Required)

Added AWS Seoul (ap-northeast-2)

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by D3Hunter — APPROVED** [review]
lgtm

**Review by qiancai — APPROVED** [review]

**Review by Frank945946 — APPROVED** [review]

File: tidb-cloud/tidb-cloud-billing-ticdc-rcu.md [hunk 1]
```diff
@@ -21,20 +21,4 @@ For each TiDB cluster, the number of TiCDC RCUs is set up by TiDB Cloud accordin
...
-| AWS            | Tokyo (ap-northeast-1)      |          $0.1669 |
-| AWS            | Frankfurt (eu-central-1)    |          $0.1564 |
-| GCP            | Oregon (us-west1)           |          $0.1452 |
-| GCP            | N. Virginia (us-east4)      |          $0.1626 |
-| GCP            | Iowa (us-central1)          |          $0.1452 |
-| GCP            | Singapore (asia-southeast1) |          $0.1746 |
-| GCP            | Taiwan (asia-east1)         |          $0.1628 |
-| GCP            | Tokyo (asia-northeast1)     |          $0.1868 |
-| GCP            | Osaka (asia-northeast2)     |          $0.1868 |
+For more information about the supported regions and the price of TiDB Cloud for each TiCDC RCUs, see [Changefeed Cost](https://www.pingcap.com/tidb-cloud-pricing-details/#changefeed-cost).
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-billing-ticdc-rcu.md:24]:
```suggestion
For more information about the supported regions and the price of TiDB Cloud for each TiCDC RCU, see [Changefeed Cost](https://www.pingcap.com/tidb-cloud-pricing-details/#changefeed-cost).
```

File: tidb-cloud/backup-and-restore.md [hunk 2]
```diff
@@ -70,7 +70,7 @@ Currently, you cannot select an arbitrary remote region for backup data storage.
 
 > **Note:**
 >
-> If you select multiple backup storage regions, you will be charged for multiple backup storage and inter-region backup data replication out from the cluster region to each destination region. The cost is on a per-region basis and varies with the backup regions selected. For more information, see [Data Backup Cost](https://en.pingcap.com/tidb-cloud-pricing-details/#data-backup-cost).
+> If you select multiple backup storage regions, you will be charged for multiple backup storage and inter-region backup data replication out from the cluster region to each destination region. The cost is on a per-region basis and varies with the backup regions selected. For more information, see [Data Backup Cost](https://www.pingcap.com/tidb-cloud-pricing-details/#backup-storage-cost).
```
Comment (qiancai) [inline tidb-cloud/backup-and-restore.md:73]: @allengaoo PTAL

**Review by Frank945946 — APPROVED** [review]

File: tidb-cloud/tidb-cloud-billing-dm.md [hunk 3]
```diff
@@ -28,16 +28,7 @@ The Data Migration job measures incremental data migration performance in rows/s
...
-
-| Cloud provider | Region                      | RCU price ($/hr) |
-|----------------|-----------------------------|------------------|
-| AWS            | Oregon (us-west-2)          |          $0.1307 |
-| AWS            | N. Virginia (us-east-1)     |          $0.1307 |
-| AWS            | Mumbai (ap-south-1)         |          $0.1393 |
-| AWS            | Singapore (ap-southeast-1)  |          $0.1623 |
-| AWS            | Tokyo (ap-northeast-1)      |          $0.1669 |
-| AWS            | Frankfurt (eu-central-1)    |          $0.1564 |
+For more information about the supported regions and the price of TiDB Cloud for each Data Migration RCU, see [Data Migration Cost](https://www.pingcap.com/tidb-cloud-pricing-details/#dm-cost).
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-billing-dm.md:31]:
```suggestion
To learn about the supported regions and the price of TiDB Cloud for each Data Migration RCU, see [Data Migration Cost](https://www.pingcap.com/tidb-cloud-pricing-details/#dm-cost).
```

**Review by ran-huang — APPROVED** [review]

File: tidb-cloud/backup-and-restore.md (hunk 2 above)
Comment (allengaoo) [reply tidb-cloud/backup-and-restore.md:73]: LGTM

File: tidb-cloud/tidb-cloud-billing-ticdc-rcu.md [hunk 4]
```diff
@@ -21,20 +21,4 @@ For each TiDB cluster, the number of TiCDC RCUs is set up by TiDB Cloud accordin
...
-| AWS            | Tokyo (ap-northeast-1)      |          $0.1669 |
-| AWS            | Frankfurt (eu-central-1)    |          $0.1564 |
-| GCP            | Oregon (us-west1)           |          $0.1452 |
-| GCP            | N. Virginia (us-east4)      |          $0.1626 |
-| GCP            | Iowa (us-central1)          |          $0.1452 |
-| GCP            | Singapore (asia-southeast1) |          $0.1746 |
-| GCP            | Taiwan (asia-east1)         |          $0.1628 |
-| GCP            | Tokyo (asia-northeast1)     |          $0.1868 |
-| GCP            | Osaka (asia-northeast2)     |          $0.1868 |
+To learn about the supported regions and the price of TiDB Cloud for each TiCDC RCU, see [Changefeed Cost](https://www.pingcap.com/tidb-cloud-pricing-details/#changefeed-cost).
```
Comment (hfxsd) [inline tidb-cloud/tidb-cloud-billing-ticdc-rcu.md:24]:
```suggestion
To learn about the supported regions and the price of TiDB Cloud for each TiCDC RCU, see [Changefeed Cost](https://www.pingcap.com/tidb-cloud-pricing-details/#changefeed-cost). 
```

**Pull request merged**